multiqc/utils/config_defaults.yaml

```diff
@@ -468,3 +468,7 @@ module_order:
             - RNA
             - DNA
             - methylation
+    - mirtop:
+        module_tag:
+            - RNA
```

Comment:
Feel free to make new tags :) I think `mirna` and `smrna` would be better here as it's quite a specific tool.